OPAL/tac/src/main/scala/org/opalj/tac/cg/android/AndroidICCAnalysis.scala

```diff
@@ -731,6 +738,15 @@ class AndroidICCAnalysis(val project: SomeProject) extends FPCFAnalysis {
         }
     }
 
+    def findString(assignment: Assignment[V]): Option[String] = {
+        val expr = assignment.expr
+        if (expr.isStringConst) {
+            Some(expr.asStringConst.value)
+        } else if (expr.isGetStatic) {
+            Some(expr.asGetStatic.name)
```

Comment:
This seems very strange. The name of a static field and the value of a String constant are usually two very different things. Are you sure that this is correct here (and that what you would need is not, e.g., the actual value stored in the field instead of the field's name)?